src/components/section/section.jsx

```diff
@@ -20,6 +20,14 @@ const SectionContent = styled.div`
         ? setWidth(config.sandboxMinWidth)
         : '320px'};
     margin: auto;
+
+    @media screen and (max-width: 375px) {
+        min-width: auto;
+    }
+
+    @media screen and (min-width: 1440px) {
+        max-width: 1440px;
```

Comment:
can we reuse setWidth(config.sandboxMaxWidth) here with fallback to 1440?

Reply:
When it possible we should separate VRT special things inside Styleguide from actual Styleguide. Those media are for Styleguide usability. VRT stylesheets should be moved to separate file.